sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1238,6 +1238,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val COMBINE_LIMIT_AFTER_SORT_THRESHOLD =
+    buildConf("spark.sql.execution.combineLimitAfterSortThreshold")
+      .internal()
+      .doc("In sql like 'select x from t order by y limit m', if m is under this threshold, " +
```

Comment:
`In sql queries with a sort followed by a limit like 'select x from t order by y limit m', if m ...`